src/main.py

```diff
@@ -45,7 +46,7 @@ def parse_args():
     return args
 
 
-@app.route('/recorded', methods=['GET', 'POST'])
+@app.route('/recorded', methods=['POST'])
 def frontend_communication_upload_recorded():
```

Comment:
We should start adding comments to important functions (as was requested on last checkpoint)

Reply:
True, we need to decide on some convention or sth